src/commonlib/option_cbfs.c

```diff
@@ -0,0 +1,48 @@
+#include <cbfs.h>
+#include <string.h>
+#include <stdint.h>
+#include <program_loading.h>
+#include <types.h>
+
+#include <commonlib/cbfs.h>
+#include <commonlib/region.h>
+#include <commonlib/cbfs_serialized.h>
+#include <console/console.h>
+
+#define BOOTORDER_FILE "bootorder"
+
+static char *after_str(const char *s, const char *pattern)
+{
+    size_t pattern_l = strlen (pattern);
```

Comment:
@BartekLew maybe we should consider input validation?

Reply:
OK, there wasn't one in the first place so I didn't do that..